### PR TITLE
Labeeba first

### PR DESCRIPTION
**Pull request by slk18bg**
last thing I worked on never got merged in; worked on shaping the final game obj, some minor things still have to be worked out

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,5 +22,6 @@
     "eslint-config-next": "14.0.1",
     "postcss": "^8",
     "tailwindcss": "^3.3.0"
-  }
+  },
+  "type": "module"
```
Comment (dotGrimmz) [inline package.json]: Did this change still need to be implemented?

File: src/pages/api/service/getGameNames.js [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
 
     // build up the query string based on these fields
     // extracting ids into easily searchable format for the query string
     // format: (1, 2, 3, 4, ...)
-    const genres = await GetGenres();
-    const platforms = await GetPlatforms();
-    const multiPlayers = await GetMultiPlayerMode();
-    const releaseDate = await GetReleaseDate();
-    const ageRating = await GetAgeRating();
+    // const genres = await GetGenres();
```
Comment (dotGrimmz) [inline src/pages/api/service/getGameNames.js]: if the commented code is not needed or duplication lets sanitize the files before merging to staging.

File: package.json (hunk 1 above)
Comment (slk18bg) [reply package.json]: Oh no, I thought I removed that. This can be left out

File: src/pages/api/service/getGameNames.js (hunk 2 above)
Comment (slk18bg) [reply src/pages/api/service/getGameNames.js]: Sounds good, I'll do that

**Review by dotGrimmz — APPROVED** [review]

**Pull request merged**